adr/query.py

```
@@ -175,8 +175,7 @@ def format_query(query, remainder=[]):
     :param name query: name of the query file to be run.
     :param remainder: user contexts
     """
-    if isinstance(config.fmt, str):
```

let the code fail, no need to protect it with this